crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
   en: "trippy"
   fr: "trippy"
   tr: "trippy"
+  pt: "trippy"
 auto:
   en: "auto"
   fr: "automatique"
   tr: "otomatik"
+  pt: "automático"
 on:
   en: "on"
   fr: "activé"
   tr: "açık"
+  pt: "ligado"
 off:
   en: "off"
   fr: "désactivé"
   tr: "kapalı"
+  pt: "desligado"
 yes:
   en: "Yes"
   fr: "Oui"
   tr: "Evet"
+  pt: "Sim"
 no:
   en: "No"
   fr: "Non"
   tr: "Hayır"
+  pt: "Não"
 none:
   en: "none"
   fr: "aucun"
   tr: "hiçbiri"
+  pt: "nenhum"
 hidden:
   en: "Hidden"
   fr: "Caché"
   tr: "Gizli"
+  pt: "Oculto"
 flow:
   en: "flow"
   fr: "flux"
   tr: "akış"
+  pt: "fluxo"
 flows:
   en: "flows"
   fr: "flux"
   tr: "akışlar"
+  pt: "fluxos"
 target:
   en: "Target"
   fr: "Cible"
   tr: "Hedef"
+  pt: "Alvo"
 config:
   en: "Config"
   fr: "Config"
   tr: "Konfigurasyon"
+  pt: "Config"
 status:
   en: "Status"
   fr: "Statut"
   tr: "Durum"
+  pt: "Estado"
 protocol:
   en: "protocol"
   fr: "protocole"
   tr: "protokol"
+  pt: "protocolo"
 as-info:
   en: "as-info"
   fr: "info-as"
   tr: "AS bilgisi"
+  pt: "info-as"
 details:
   en: "details"
   fr: "détails"
   tr: "ayrıntılar"
+  pt: "detalhes"
 max-hosts:
   en: "max-hosts"
   fr: "nombre d'hôtes maximum"
   tr: "maksimum hostlar"
+  pt: "max-hosts"
```

Comment:
You can shorten it to "max-hosts" in Portuguese in order to say "maximum number of hosts". The Portuguese translation for the longer text would be "máximo número de hosts".

Reply:
I'm trying to get rid of this one entirely as it is very long and many languages and also very unclear to users what it even means!